### ISSUE TITLE
Enable sourceMaps?

### ISSUE THREAD

**Issue by cbetta** [rollup/rollup-starter-project#8]
Do you happen to know if it's possible to enable sourcemaps with this setup? I keep getting this error:

```
Cannot generate a sourcemap if non-sourcemap-generating transformers are used
```

**Comment by silvestreh** [comment]
Was this solved by PR #6? If so someone should probably close this issue 😄

**Comment by eventualbuddha** [comment]
Yes, source maps should be on now.
